classes.js

```diff
@@ -0,0 +1,97 @@
+const bookForm = document.querySelector('#book-form');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const booksList = document.querySelector('#book-list');
+
+// create a class book with add remove add display methods
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    const bookObj = { allbook: [] };
+    if (JSON.parse(localStorage.getItem('books')) == null) {
+      localStorage.setItem('books', JSON.stringify(bookObj));
+    }
+
+    const obj = JSON.parse(localStorage.getItem('books'));
+    // set obj to empty
+    obj.allbook = [];
+    // push new book to obj
+
+    if (this.title.value !== '' && this.author.value !== '') {
+      alert('successfully added a book');
+      obj.allbook.push({
+        title: this.title,
+        author: this.author,
+      });
+    }
+    localStorage.setItem('books', JSON.stringify(obj));
+  }
```

Comment:
- [ ] Kindly check your book addition. Adding the first book, when I now add the second one it removes the first book from the local storage and places only the second book. See image below. You can see I have two books on the page, but only the last one is stored in the local storage.

  ![saw](https://user-images.githubusercontent.com/81468675/191730041-7958c275-8fce-450b-8963-dd5c949f39a3.jpg)

Reply:
thank you i solved the issue